### DISCUSSION TITLE
Mastodon pwa doesn't work properly issue #838

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1575] [General]
Well, today I found out that mastodon pwa won't work properly, and I already found an issue that is created #838 but didnt quite understand what is going on despite reading the issue. So, is this a problem about a bromite, mastodon or what? Is it going to be fixed?
It works fine on chromium but I get black screen when I use mastodon pwa on bromite. Latest Bromite ARM version MIUI.

**Comment by csagan5** [comment]
It is probably related to this patch: https://github.com/bromite/bromite/blob/master/build/patches/Enable-network-isolation-features.patch

Someone would have to verify this then submit an issue/PR; nobody has submitted a minimal testcase that does not involve third-party servers so there is no open issue about it.

Feel free to proceed as above (verify the problem and/or submit a minimal testcase); asking someone else to do this is unfortunately not an option.

**Reply by ghost** [reply]
Thanks for informing.
